zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/ObjectenApiPlugin.kt

```diff
@@ -33,13 +35,29 @@ class ObjectenApiPlugin(
 ) {
     @PluginProperty(key = "url", secret = false)
     lateinit var url: URI
+
     @PluginProperty(key = "authenticationPluginConfiguration", secret = false)
     lateinit var authenticationPluginConfiguration: ObjectenApiAuthentication
 
     fun getObject(objectUrl: URI): ObjectWrapper {
         return objectenApiClient.getObject(authenticationPluginConfiguration, objectUrl)
     }
 
+    fun getObjectsByObjectTypeId(
+        objecttypesApiUrl: URI,
+        objectsApiUrl: URI,
+        objecttypeId: String,
+        pageable: Pageable
+    ): ObjectsList {
+        return objectenApiClient.getObjectsByObjecttypeUrl(
+            authenticationPluginConfiguration,
+            objecttypesApiUrl,
+            objectsApiUrl,
+            objecttypeId,
+            pageable
```

Comment:
how is the pageable constructed based on the output of this method? the next and previous (page) variables are present in the response, but i don't really see a way of using it to request the next page.